util/translate_fight.py

```diff
@@ -181,6 +183,14 @@ def main(args):
     for lang in languages:
         if lang == default_language:
             continue
+        # Fix lower mob names like "ao-no-shiki" to "Aka-No-Shiki" and "Aka-no-shiki"
```

Comment:
I think this does not happen frequently enough to do for all abilities with space or hyphen in the name, and is better to just do manually.  Are you thinking of other examples other than just things with "no" in the name?

I do feel like there are continual french issues (e.g. ecbb723f097328c7bd0476352e5135bd5f776248) and so at some point it might be worth making everything case insensitive.

Reply:
I think most words with space or hyphen in Chinese logs have this problem, like `dark crystal` of o9s, `ancient dragon` and `immortal key` in o10s and so on, so maybe case insensitive is a good solution.

Reply:
Lots of mobs have spaces in their name, so this seems like it would generate a lot of entries.

I think I would prefer to remove this section and to leave just the original name (which hopefully matches the one in the timeline).  The timelineReplace section has two purposes: (1) it's used in **util/check_translation.py** to automatically add triggers in other languages and (2) it's used at runtime to modify the timeline names and syncs.

If you have a lot of names that don't match, then what about this proposal:
* leave the original mob name in the timelineReplace section
* modify check_translation.py to be case insensitive when doing replacements
* maybe make regexCn / regexFr in triggers be case insensitive by making them be `/regex/i` instead of just `/regex/`

Reply:
Okay I’ll try to make it to /regex/i when I get back to work